python/dgl/graphbolt/itemset.py

```diff
@@ -1,7 +1,7 @@
 """GraphBolt Itemset."""
 
 import textwrap
-from typing import Dict, Iterable, Iterator, Tuple, Union
+from typing import Dict, Iterable, Sequence, Tuple, Union
```

Comment:
Looks like typing.Sequence is deprecated.

https://docs.python.org/3/library/typing.html#typing.Sequence

Reply:
Use https://docs.python.org/3/library/collections.abc.html#collections.abc.Sequence instead.

Reply:
Looks like most of the things in typing are deprecated, we need to replace them with their alternatives.

Reply:
Could you open an issue to track this?

Reply:
Thank you for letting me know this.

Reply:
See https://github.com/dmlc/dgl/issues/7400 . But for now, let's keep them because "the aliases are guaranteed to remain in the typing module without deprecation warnings until at least Python 3.14". Quoted from https://docs.python.org/3/library/typing.html#deprecated-aliases .